libultraship/libultraship/Window.cpp

```diff
@@ -347,8 +348,9 @@ namespace Ship {
         lastScancode = -1;
 
         bool bIsProcessed = false;
-        const auto pad = dynamic_cast<KeyboardController*>(ControllerApi->physicalDevices[ControllerApi->physicalDevices.size() - 2].get());
-        if (pad != nullptr) {
+        auto controlDeck = GlobalCtx2::GetInstance()->GetWindow()->GetControlDeck();
+        const auto pad = dynamic_cast<KeyboardController*>(controlDeck->GetPhysicalDevice(controlDeck->GetNumPhysicalDevices() - 2).get());
+    	if (pad != nullptr) {
```

Comment:
looks like there's a tab mixed in spaces here, seems the rest of the file is using spaces